lib/fhc.js

```diff
@@ -152,7 +152,7 @@ fhc.applyCommandFunction = function (processArgv, cb) {
   // Only new-style commands can be validated
   if (cmdFn.demand) {
     try {
-      argUtils.validate(yargs, cmdFn);
+      argv = argUtils.validate(yargs, cmdFn);
```

Comment:
@camilamacedo86 only the third one is not supported. It is not the right way to specify aliases - it has to be prefixed with "-" or "--"